modules/openshift-developer-cli-installing-odo-on-linux.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/installing-odo.adoc     
+
+[id="installing-odo-on-linux"]
+
+= Installing {odo-title} on Linux
+
+== RPM installation
+* For CentOS or RHEL use:
++
+----
+$ yum install odo
+----
+* For Fedora use:
++
+----
+$ dnf install odo
+----
+
+== Binary installation
+
+----
+# curl -L https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-darwin-amd64 -o /usr/local/bin/odo && chmod +x /usr/local/bin/odo
+----
+
+== Tarball installation
+
+----
+# sh -c 'curl -L https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-linux-amd64.tar.gz | gzip -d > /usr/local/bin/odo; chmod +x /usr/local/bin/odo'
```

Comment:
Similar to above comment, should be multiple lines as well as using `$` too.

Maybe put a note saying it needs to be using `root` to install?

Reply:
```sh
$ sh -c 'curl -L https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-linux-amd64.tar.gz | gzip -d > /usr/local/bin/odo
$ chmod +x /usr/local/bin/odo'
```